README.md

```diff
@@ -1,69 +1,107 @@
-# get-starknet<br/>StarkNet wallet <-> dApp bridge
+# get-starknet
 
 [![npm](https://img.shields.io/npm/v/get-starknet.svg)](https://www.npmjs.com/package/get-starknet)
 
-## Alpha version
-
 ## Goals
 
-- Allow dApps to seamlessly connect to any wallet on StarkNet
-- Allow wallets to seamlessly connect to any dApp, and get discovered by users which need a new wallet
-- An open source wallet/dApp API controlled by the community
-- An open source StarkNet wallet discovery list moderated by the community
+- ❤️‍🩹 Allow Starknet dApps and wallets to seamlessly connect
+- 🪶 Lightweight and easy to use
+- 🏎 Fast integration, building and testing
+- ⚙️ Customizable and extensible
+- 🌍 Open source and controlled by the community
 
 ## Installation
+
 ```
 # using npm
-npm install get-starknet
+npm install get-starknet starknet@next
 
 # using yarn
-yarn add get-starknet
+yarn add get-starknet starknet@next
+
+# using pnpm
+pnpm add get-starknet starknet@next
 ```
 
 ## Usage for dApp developers
 
-If you were using getStarknet() before, simply replace the import line as below.
```

Comment:
let's keep it, it's an important migration hint.

Reply:
v2 aims to simplify the library. With a breaking change (starknet.js v4) we have the opportunity to do more breaking changes, so I decided to remove `getStarknet` from the library. I think there should only exist one method to achieve the same thing.

Let me know if you would prefer to keep it

Reply:
but `getStarknet` is still available + exported on this branch (`packages/core/src/main.ts:63`);
if we keep it - then I think we should keep the migration hint, and if we don't - let's remove it from the branch completely 😄

Reply:
it's a completely different function now

If develop `get-starknet` they will install the UI package. Just those who choose to pick `core` will be able to access it. So it won't be an issue for backwards compatability, as there is none.

Reply:
ok